package.json

```diff
@@ -21,10 +21,10 @@
     "postrewrite": "yarn git-hook",
     "precommit": "yarn git-hook",
     "prepush": "yarn git-hook && yarn snyk test",
-    "start": "yarn build-header && node app.js",
-    "start-watch": "yarn build-header && yarn brunch watch & nodemon app.js | ./node_modules/bunyan/bin/bunyan",
-    "test": "yarn build-header && NODE_ENV=test mocha --exit --recursive test",
-    "test-ci": "yarn build-header && yarn ci-hook && yarn coverage-upload-coveralls",
+    "start": "yarn header-build && node app.js",
```

Comment:
Does `header-build` need to be run here given it is running from the `Dockerfile` and going through the `Dockerfile` is the only way the application is going to start up correctly.

Reply:
without the header-build in the task this won't work locally if the 'code' folder is mounted.
Start is not used in the container, the app is started directly to avoid multiple thread issues.

Reply:
Good point about how the container starts.
Is the `start` script used for anything? Could it be removed?

Reply:
It can go, it's not used anywhere. I'll remove it.